feat: START add support to hide action

hitendreamworld opened the pull request.

hitendreamworld committed:
feat: START add support to hide action [ci skip]

ChiragMoradiya commented on dw-menu.js:
```diff
@@ -126,34 +127,37 @@ export class DwMenu extends DwCompositeDialog {
-            ? html`<span id=${action.name}>
-                <dw-list-item
+      ${repeat(
+        this.actions.filter((action) => this.hiddenActions.indexOf(action.name) === -1),
```
Create a private method `_isItemHidden` for better readability.

hitendreamworld committed:
feat: rework [ci skip]

- Create a private method `_getActions` for filter actions
- update documentation
- add sample images.
- fix: if tip is not available tooltip, do not render tooltip

dw-devops commented:
:tada: This PR is included in version 1.1.0 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/@dreamworld/dw-menu/v/1.1.0)
- [GitHub release](https://github.com/DreamworldSolutions/dw-menu/releases/tag/v1.1.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
